.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "python.pythonPath": "${env:HOME}/miniconda3/envs/pytorch/bin/python",
```

Comment:
This line should probably be removed, since e.g. people might have a different name for their `conda` environment, or not use one at all.
```suggestion
```

Reply:
was about to say that assumes anything from the environment (for example conda instead of pip virtual env, and the name of the conda env is `pytorch`).